osu.Game.Rulesets.Pippidon/UI/PippidonPlayfield.cs

```diff
@@ -117,16 +118,17 @@ private set
 
             public Texture Texture
             {
-                set
-                {
-                    ((Sprite)Child).Texture = value;
-                }
+                set { ((Sprite)Child).Texture = value; }

```

Comment:
`set => ...`?

Reply:
feel free to change as you see fit. if you're going to start complaining about code styling, there's a *lot* more that needs fixing across this whole codebase :P

Reply:
you're probably right :P

Reply:
you're probably right :P